package.json

```diff
@@ -7,10 +7,12 @@
   "license": "MIT",
   "repository": "https://github.com/Axinom/drm-quick-start",
   "dependencies": {
+    "commander": "^2.9.0",
```

Comment:
I understand devDependencies are those, which shouldn't required for "normal" use. More like extra modules for executing tests, creating/combining new minified packages and such.